soh/soh/Enhancements/randomizer/3drando/settings.cpp

```diff
@@ -2475,7 +2475,15 @@ namespace Settings {
     BridgeDungeonCount.SetSelectedIndex(cvarSettings[RSK_RAINBOW_BRIDGE_DUNGEON_COUNT]);
     BridgeTokenCount.SetSelectedIndex(cvarSettings[RSK_RAINBOW_BRIDGE_TOKEN_COUNT]);
     RandomGanonsTrials.SetSelectedIndex(cvarSettings[RSK_RANDOM_TRIALS]);
-    GanonsTrialsCount.SetSelectedIndex(cvarSettings[RSK_TRIAL_COUNT]);
+    //GanonsTrialsCount.SetSelectedIndex(cvarSettings[RSK_TRIAL_COUNT]);
```

Comment:
might be nice to add a `RANDOTODO` here and next to the other commented out block saying we want to bring back the slider and implement the specific trial requirements for the barrier